stl/inc/format

```diff
@@ -2995,6 +2996,13 @@ private:
     }
 
 public:
+    _NODISCARD bool operator==(default_sentinel_t) const noexcept {
+        return _First >= _Last;

```

Comment:
In what case would `_First > _Last`?

Reply:
I believe that would be an invariant violation - we control this internal type so we should never allow `_First` to move beyond `_Last`. I'll change this to equality.